openmdao/utils/testing_utils.py

```diff
@@ -247,6 +248,28 @@ def set_env_vars_context(**kwargs):
                 os.environ[k] = v
 
 
+@set_env_vars(OPENMDAO_CI="1")
+def force_check_partials(prob, *args, **kwargs):
+    r"""
+    Force the checking of partials even for components with _no_check_partials set.
+
```

Comment:
It occurs to me that maybe the env var should be more specific, e.g. `FORCE_CHECK_PARTIALS` since that's the one thing it's used for..  What do you think?

Reply:
I agree with @swryan here.  Undoubtedly in the future we'll do something else that needs a CI flag but a more descriptive name would be useful here.

Reply:
fixed